tests/runners/analysis_submission_runner.py

```diff
@@ -88,18 +88,25 @@ def create_analysis_submission(self):
         self.analysis_submission = self.ingest_api.envelope(envelope_id=None, url=submission_url)
         process = self.ingest_client_api.create_entity(submission_url, self.analysis_fixture.analysis_process, 'processes')
         protocol = self.ingest_client_api.create_entity(submission_url, self.analysis_fixture.analysis_protocol, 'protocols')
+        input_files = [self.ingest_client_api.create_file(submission_url, file["file_core"]["file_name"], file) for file in self.analysis_fixture.input_files]
```

Comment:
why not just get the files from the first submission?

Reply:
I tried that originally, it doesn't appear to have files